app/views/public/schedule/style.css.erb

```diff
@@ -103,5 +103,5 @@
 /* Custom Conference CSS */
 
 <% if @conference.schedule_custom_css.present? %>
-  <%= @conference.schedule_custom_css.gsub(/\{([^}]*)\}/m){ '{' + sanitize_css($1) + '}'; } %>
```

Comment:
As I said, the code as it stands tampers with perfectly valid, non-malicious content. So if frab wants to stick with this kind of sanitizing, it needs to be improved a lot and replaced by something that actually understands the semantics of CSS.

I'll keep that commit in my tree as that's currently the easiest way to make frab to what I want it to.